mama_ng_control/apps/vumimessages/models.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+
+from django.contrib.postgres.fields import HStoreField
+from django.db import models
+
+from mama_ng_control.apps.contacts.models import Contact
+
+
+class Outbound(models.Model):
+
+    """
+    Contacts outbound messages and their status
+    """
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    contact = models.ForeignKey(Contact,
+                                related_name='messages',
+                                null=False)
+    version = models.IntegerField(default=1)
+    content = models.CharField(null=True, blank=True, max_length=1000)
```

Comment:
No, it's designed to be audio files only - but I put this in just in case there's a text-to-speech need sometime